pages/learn/accounts/enterprise-sso.md

```diff
@@ -0,0 +1,118 @@
+
+This identifier must be unique within balenaCloud and can only include lowercase letters (`a-z`), numbers (`0-9`), hyphens (`-`), and underscores (`_`). 
+
+### Associate Organizations and Teams
+
+BalenaCloud would need a list of organizations to which SAML users will be automatically added upon successful authentication. Only organizations subscribed to an [Enterprise plan](https://www.balena.io/) will appear in the list of available organizations. At least one organization must be provided when setting up the IdP.
+
+__Note:__ Removing organizations after IdP creation will not revoke access for SAML users who have previously authenticated with this IdP. However, new authentications will no longer include the removed organization. An IdP will always require at least one organization to be associated.
+
+You have successfully configured SAML 2.0 for your balenaCloud Enterprise SSO. Your team can now access the platform securely and seamlessly through the configured Identity Provider. For instructions on how your team can log in, refer to [Authenticating as a SAML/SSO User](#authenticating-as-a-samlsso-user). If you encounter any issues or need further assistance, please contact our support team.
+<!-- TODO: Update to latest screenshot -->
+<img alt="Fully configured IdP with two Orgs associated" src="/img/common/saml/idp-with-two-orgs.png" width="100%">
+
+#### (Optional) Configure a Default Team
+
+You can configure a default team for each organization to which SAML users will be automatically added upon authentication.
+
+__Note__: If you unlink the default team in the Identity Provider configuration, it will not remove team access for SAML users who have already authenticated. However, new SAML users will no longer be automatically added to this team.
+
+## Authenticating as a SAML User
+
+### Setting up a new SAML user
+
+To log in using your enterprise SAML authentication, you must first have or [create](https://dashboard.balena-cloud.com/signup) a standard balenaCloud account using your company email address. Once logged in to this account, navigate to your [user preferences](https://dashboard.balena-cloud.com/preferences/details) and click "Enable" in the "Enterprise SSO" section.
+<!-- TODO: Update to latest screenshot -->
+<img alt="User preferences with enable SSO button highlighted" src="/img/common/saml/merge-account-accept.png" width="100%">
+
+Next, provide the company `SSO Identifier` supplied by your balenaCloud organization administrator.
+<!-- TODO: Update to latest screenshot -->
+<img alt="Enable SSO modal with SSO identifier filled in." src="/img/common/saml/add-sso-identifier-merge-modal.png" width="60%">
+
+__Important:__ By activating SAML, you are transferring your personal account to a company account. The following changes will occur:
```

Comment:
We should probably add a comment mentioning that this is non (at least trivially) roll back, e.g. once a user becomes saml managed it can't be easily undone (it will require intervention from us)